index.html

```diff
@@ -0,0 +1,974 @@
+                </ul>
+            </nav>
+
+            <nav class="footer-top-nav">
+                <h3 class="other-nav-heading">Products</h3> 
+                    <ul class="company-list">
+                        <li class="company-item">
+                            <a href="catalog.html" class="top-nav-link">Website</a>
+                        </li>
+
+                        <li class="company-item">
+                            <a href="#" class="top-nav-link">Mobile App</a>
+                        </li>
+
+                        <li class="company-item">
+                            <a href="#" class="top-nav-link">Social Media Feeds</a>
+                        </li>
+
+                        <li class="company-item">
+                            <a href="#" class="top-nav-link">Presentation</a>
+                        </li>
+
+                        <li class="company-item">
+                            <a href="#" class="top-nav-link">Product Mockups</a>
+                        </li>
+
+                        <li class="company-item">
+                            <a href="#" class="top-nav-link">Illustrations</a>
+                        </li>
+                    </ul>
+            </nav>
+
+            <nav class="footer-nav-company">
+                <h3 class="other-nav-heading">Account</h3> 
+                    <ul class="company-list">
+                        <li class="company-item">
+                            <a href="cart.html" class="top-nav-link">You Cart</a>
+                        </li>
+
+                        <li class="company-item">
+                            <a href="#" class="top-nav-link">Check Out</a>
+                        </li>
+
+                        <li class="company-item">
+                            <a href="#" class="top-nav-link">Refund Policy</a>
+                        </li>
+
+                        <li class="company-item">
+                            <a href="#" class="top-nav-link">Company Policy</a>
+                        </li>
+
+                        <li class="company-item">
+                            <a href="#" class="top-nav-link">Send A Message</a>
+                        </li>
+
+                        <li class="company-item">
+                            <a href="#" class="top-nav-link">Customer Support</a>
+                        </li>
+                    </ul>
+            </nav>
+
+            <form action="/user-email-newsletter" method="POST" class="subscribtion-form"> 
+                <label for="email" class="subscribtion-form-heading">Subscribe to Newsletter</label> 
+
+                <input type="email" name="email" placeholder="Enter your email address ..." required class="subscribtion-form-input">
+
+                <button type="submit" class="subscribtion-form-button">Subscribe</button>
+            </form>
+        </section>
+
+        <section class="social-media">
+                <h2 class="social-media-heading">Follow Us</h2>
```

Comment:
You have something like a double indent here and in a section below, check, please